app/assets/stylesheets/layout/_how-it-works.scss

```diff
@@ -0,0 +1,16 @@
+.how-it-works__inf {
+  text-align: center;
+}
+
+.how-it-works__category {
+  @media (min-width: 768px) {
```

Comment:
This should be a variable defined in `abstracts/_variables.scss` this size could be `$small-screen`. What do you think?